bin/celery_worker.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
`docker_start.sh` still uses sh too, but we can actually use `bash` instead of `sh` since Debian is used as base image :)